lib/tasks/publishing_api.rake

```diff
@@ -188,6 +188,18 @@ namespace :publishing_api do
     puts "Finished queuing items for Publishing API"
   end
 
+  desc "Republish all documents with official attachments"
+  task republish_documents_with_official_attachments: :environment do
+    attachment_ids = Attachment.where("unnumbered_command_paper = true OR command_paper_number != '' OR unnumbered_hoc_paper = true OR hoc_paper_number != ''").pluck(:attachable_id).uniq
+    document_ids = Document.joins(:editions).where("editions.id": attachment_ids).pluck(:document_id)
```

Comment:
Could this list have duplicate document ids in it?

Reply:
yeah, almost certainly good spot, i'll put `.uniq` on it

Reply:
just checked on integration and that's cut 4000 documents, thanks!